libminifi/src/SchedulingAgent.cpp

```diff
@@ -40,7 +40,7 @@ bool SchedulingAgent::hasWorkToDo(std::shared_ptr<core::Processor> processor) {
 }
 
 std::future<bool> SchedulingAgent::enableControllerService(std::shared_ptr<core::controller::ControllerServiceNode> &serviceNode) {
-  logger_->log_info("Enabling CSN in SchedulingAgent %s", serviceNode->getName());
```

Comment:
As per my comment below I think these are candidates for INFO based on the belief that support any type of teir 1 support might need this information.